addons/product/models/res_config_settings.py

```diff
@@ -79,3 +74,8 @@ def _onchange_module_module_sale_product_matrix(self):
         If the user enables the Grid Configurator -> enable the product Configurator as well"""
         if self.module_sale_product_matrix and not self.module_sale_product_configurator:
             self.module_sale_product_configurator = True
+
+    def _toggle_group_multi_currency(self):
+        super()._toggle_group_multi_currency()
+        if self.group_multi_currency:
+            self.group_product_pricelist = True
```

Comment:
You're setting a field in an api.model method, it will crash or won't have any impact (because self is empty)

Reply:
Can you tell me why it is working? I just tested on runbot now, and group_product_pricelist is activated

Reply:
Because when `sale` is installed, `product.group_product_pricelist` is an implied group of `base.group_multi_currency`

```xml
    <record id="base.group_multi_currency" model="res.groups">
        <!--
            Sales multi-currency is based on pricelists, if multi-currency is enabled,
            enable pricelists.
        -->
        <field name="implied_ids" eval="[(4, ref('product.group_product_pricelist'))]"/>
    </record>
```

Reply:
I was just doing the very same in the product module, can we not move this to product and remove the need to override `_toggle_group_multi_currency` ? @smetl

Reply:
Due to 7d8df4b - the implied group should move to product, that way POS can benefit too.